docs/learn/concepts.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_label: '📑 Major Concepts'
+sidebar_position: 2
+---
+
+# Major Concepts
+
+## Universal Profile (UP):
+
+A Universal Profile (UP) is a smart contract-based account that acts as the center for all your blockchain interactions. UPs are generic for various use cases, such as regular wallets, and can interact directly with onchain applications. Additionally, they also feature permission and asset management, updatable profile information, and let you interact as one persona through different controller keys.
+
+## Controller:
+
+In the [LSP6 Key Manager](../standards/universal-profile/lsp6-key-manager.md), the term "controller" refers to a permissioned address. These addresses have permissions that allow them to perform certain actions on the [LSP0ERC725Account](../standards/universal-profile/lsp0-erc725account.md) linked with the Key Manager (e.g., setting data or transferring LYX from the account).
```

Comment:
```suggestion
In the [LSP6 Key Manager](../standards/universal-profile/lsp6-key-manager.md), the term _"controller"_ refers to a permissioned address. These addresses have permissions that allow them to perform certain actions on the [LSP0ERC725Account](../standards/universal-profile/lsp0-erc725account.md) linked with the Key Manager (e.g., setting data or transferring LYX from the account).
```

Reply:
@biancabuzea200 This definition is also in the LSP6 page. Do you think it is necessary to have it in both places in this case?

<img width="1029" alt="image" src="https://github.com/lukso-network/docs/assets/31145285/36643f93-19df-48cb-aa9f-08c53a274ec2">

Reply:
Decided to keep duplication in both pages